src/EventStore.Core/Messages/ClientMessage.cs

```diff
@@ -1488,15 +1488,26 @@ public class ScavengeDatabase : Message {
 			public readonly IPrincipal User;
 			public readonly int StartFromChunk;
 			public readonly int Threads;
+			public readonly int? Threshold;
```

Comment:
it could be interesting to add a `bool SyncOnly` option, which would not create any scavenge points but would only run a scavenge up to the latest existing scavenge point. after doing a normal scavenge, the other nodes of the cluster could run a scavenge with `SyncOnly = true` so that all the nodes have a consistent list of chunks.

Reply:
done

Reply:
great!